Libraries/rocSPARSE/level_2/bsrxmv/bsrxmv_vs2017.vcxproj

```diff
@@ -15,7 +15,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{CFA1826D-C202-4F97-92C9-DB019675DCCB}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <RootNamespace>bsrmv_vs2017</RootNamespace>
```

Comment:
Thanks for catching this. Nonetheless - to a separate `bsr(x)mv` PR.

Reply:
Done!